app/views/govuk_publishing_components/components/_google_tag_manager_script.html.erb

```diff
@@ -8,6 +8,10 @@
   gtm_attributes = gtm_attributes.join('&')
 %>
 <script>
+dataLayer = [
```

Comment:
Just want to check - should this be `window.dataLayer` or is this something different?

Reply:
To be honest I don't fully understand the distinction - they appear to be synonymous 🤷 

Here's some console output...

<img width="763" alt="Screenshot 2022-09-01 at 08 56 23" src="https://user-images.githubusercontent.com/44037625/187862726-61cb1f9e-05f8-44c6-b030-7c624369f8db.png">